indexer/services/roundtable/src/tasks/update-compliance-data.ts

```diff
@@ -318,10 +319,18 @@ async function getComplianceData(
         return result.value;
       },
     ));
+    const responses404:
+    PromiseFulfilledResult<ComplianceClientResponse>[] = successResponses.filter(
+      (result: PromiseSettledResult<ComplianceClientResponse>):
+      result is PromiseFulfilledResult<ComplianceClientResponse> => {
+        return result.status === 'fulfilled' && result.value.riskScore === NOT_IN_BLOCKCHAIN_RISK_SCORE.toString();
```

Comment:
this logic is kinda messy, up to you, but would it be worthwhile to further populate `ComplianceClientResponse`? We already support return an `undefined` riskScore, might be cleaner to add a `status` field and have that encompass all of the states of `blocked`, `NOT_IN_BLOCKCHAIN`, and `NO_RULES_TRIGGERED`

Reply:
Yes i was actually thinking of adding a new field to CompliacenClientResponse to track the states.

Reply:
Looked further into it. Modifying ComplianceClientResponse has ripple effects into all other providers. We can leave the new field nullable. At minimum the mocks for elliptic provider need to be changed. Will just stick to conditioning on riskScore and add comment explaining.